From: Clint Byrum <clint@ubuntu.com>
Description: test fails on buildds. Apparently a race condition in the test
 itself is more reliably triggered on the buildds. Confirmed with Kapil
 that its not a real error, so the test can be disabled until there is a
 fix upstream.
Forwarded: N/A

Index: juju-0.5+bzr398/juju/agents/tests/test_unit.py
===================================================================
--- juju-0.5+bzr398.orig/juju/agents/tests/test_unit.py	2011-10-05 06:59:44.000000000 -0700
+++ juju-0.5+bzr398/juju/agents/tests/test_unit.py	2011-10-09 09:44:18.759824747 -0700
@@ -236,7 +236,7 @@
                          wordpress_states["unit"].unit_name)
 
     @inlineCallbacks
-    def test_agent_executes_config_changed_hook(self):
+    def _test_agent_executes_config_changed_hook(self):
         """Service config changes fire a config-changed hook."""
         self.agent.set_watch_enabled(True)
         self.write_empty_hooks()
